stylix/palette.nix

```
@@ -10,21 +10,49 @@ let
   opts = options.stylix;
 in
 {
+  imports = lib.singleton (
+    lib.mkRenamedOptionModule
+      [ "stylix" "polarity" ]
+      [ "stylix" "colorGeneration" "polarity" ]
+  );
   options.stylix = {
-    polarity = lib.mkOption {
-      type = lib.types.enum [
-        "either"
-        "light"
-        "dark"
-      ];
-      default = "either";
-      description = ''
-        Use this option to force a light or dark theme.
+    colorGeneration = {
+      scheme = lib.mkOption {
+        type = lib.types.enum [
+          "content"
+          "expressive"
+          "fidelity"
+          "fruit-salad"
+          "monochrome"
+          "neutral"
+          "rainbow"
+          "tonal-spot"
+          "vibrant"
+        ];
+        default = "content";
```

What about using Matugen's default as an example:

```suggestion
        default = "content";
        example = "tonal-spot";
```